src/caffe/proto/caffe.proto

```diff
@@ -30,6 +30,10 @@ message Datum {
   optional bool encoded = 7 [default = false];
 }
 
+message DatumVector {
```

Comment:
Seconding @sguada. Note also that there is a BlobProtoVector that could be
used for packing multiple pieces of data too.

On Sun, Feb 22, 2015 at 4:01 PM, Sergio Guadarrama <notifications@github.com
> wrote:

> In src/caffe/proto/caffe.proto
> <https://github.com/BVLC/caffe/pull/1942#discussion_r25138930>:
>
> > @@ -30,6 +30,10 @@ message Datum {
> >    optional bool encoded = 7 [default = false];
> >  }
> >
> > +message DatumVector {
>
> Leave that for another PR, where it is used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BVLC/caffe/pull/1942/files#r25138930>.
>